resources/tests/test_admin_actions.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+import pytest
+from django.contrib.auth import get_user_model
+
+from resources.models import Reservation
+from users.admin import anonymize_user_data
+
+
+@pytest.mark.django_db
+def test_anonymize_user_data(api_client, resource_in_unit, user):
+    """
+    Test anonymization of user data.
+    """
+    user.first_name = 'testi_ukkeli'
+    user.save()
+    Reservation.objects.create(
+        resource=resource_in_unit,
+        begin='2015-04-04T09:00:00+02:00',
+        end='2015-04-04T10:00:00+02:00',
+        user=user,
+        reserver_name='John Smith',
+        event_subject="John's welcome party",
```

Comment:
Double quotation mark `"` used instead of `'`. Should be consistent.

Reply:
This is an exception due to single quote being used inside the string.

Reply:
I think escaping the single quote would be more consistent approach. 

`'John\'s welcome party'`